extensions/ql-vscode/src/remote-queries/view/RemoteQueries.tsx

```diff
@@ -1,15 +1,112 @@
 import * as React from 'react';
 import * as Rdom from 'react-dom';
+import * as octicons from '../../view/octicons';
```

Comment:
I know this is what we have been using for a while, but could you verify that the octicons look fine in both light and dark modes?